core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -53,7 +57,7 @@
   private MetricRegistry mMetricRegistry = new MetricRegistry();
   private MetricsConfig mMetricsConfig;
   private boolean mRunning = false;
-  private MetricsServlet mMetricsServlet;
+  private Sink mMetricsServlet;
```

Comment:
It seems like the users of `getMetricsServlet` still want it as a `MetricsServlet`, would it be possible to move `MetricsServlet` into common and use that type?